modules/mobile/config/routes.rb

```diff
@@ -31,6 +31,10 @@
     get '/facilities-info/:sort', to: 'facilities_info#schedulable'
     get '/health/immunizations', to: 'immunizations#index'
     get '/health/locations/:id', to: 'locations#show'
+    scope :health do
+      resources :labs_and_tests, only: %i[index], defaults: { format: :json }
```

Comment:
I think we've been trying to switch to dashes instead of underscores.

Reply:
Yup, forgot that routing this way only produces underscore